onnxruntime/contrib_ops/cpu/transformers/subgraph_t5_decoder.cc

```diff
@@ -49,11 +49,12 @@ namespace transformers {
 
 Status T5DecoderSubgraph::Validate(const std::vector<const NodeArg*>& subgraph_inputs,
                                    const std::vector<const NodeArg*>& subgraph_outputs) {
-  bool has_hidden_state = subgraph_inputs[2]->Name() == "encoder_hidden_states" ? true : false;
-  SetPastInputIndex(has_hidden_state);
+  bool has_encoder_input_ids = subgraph_inputs[1]->Name() == "encoder_input_ids";
```

Comment:
Recommend to add a comment about example inputs:
input_ids, encoder_input_ids (optional), encoder_attention_mask, encoder_hidden_states (optional),
past_self_key_0, past_self_value_0, past_cross_key_0, past_cross_value_0,
past_self_key_1, past_self_value_1, past_cross_key_1, past_cross_value_1

Reply:
updating the comment above